### ISSUE TITLE
Add Nexo transaction import

### ISSUE THREAD

**Issue by davidhildebrand** [rotki/rotki#2224]
[Nexo](https://nexo.io/) does not yet offer an API despite frequent requests. It does currently permit CSV exports.  This is a feature request for a custom import option of the Nexo CSV files into Rotki.

CSV exports (currently) have the following formatting:
`TransactionID,Type,Currency,Amount,Description,OutstandingLoanValue,DateTime`

As far as I can tell at quick glance:
- TransactionID looks like an internal identifier.
- Type includes 'Deposit', 'Withdrawal', 'Interest', 'Dividend'.
- Currency is the three- or four-letter code for (e.g. BTC, ETH, USDC).
- Amount is the value of the transaction/transfer.
- Description includes additional details such as 'approved / BTC Interest Earned' or 'approved / blockchain_tx_hash'.
- OutstandingLoanValue seems to be the value of any outstanding loans borrowed against that currency, though I don't have access to examples.
- DateTime is the associated date and time string in the format '2021-01-01 01:00:00'.

_Originally posted by @davidhildebrand in https://github.com/rotki/rotki/issues/1674#issuecomment-770259996_

**Comment by yabirgb** [comment]
Hello @davidhildebrand We have implemented this feature but seems that the information got outdated. A user reported that the columns for the CSV file are 

`Transaction,Type,Currency,Amount,USD Equivalent,Details,Outstanding Loan,Date / Time` 

Can you confirm us this information?

**Comment by davidhildebrand** [comment]
> Hello @davidhildebrand We have implemented this feature but seems that the information got outdated. A user reported that the columns for the CSV file are
> 
> `Transaction,Type,Currency,Amount,USD Equivalent,Details,Outstanding Loan,Date / Time`
> 
> Can you confirm us this information?

Yes, this now appears to be correct.  Hopefully Nexo keeps it the same from here on out or (even better) creates an API.
Thanks for looking into this!
